Library/Homebrew/cleaner.rb

```diff
@@ -88,10 +88,17 @@ def clean_dir(d)
 
       Find.prune if @f.skip_clean? path
 
-      next if path.symlink? || path.directory?
+      next if path.directory?
 
       if path.extname == ".la"
         path.unlink
+      elsif path.symlink?
+        # Skip it.
+        #
+        # (Note that the case where the symbolic link has no extension or a
+        # different one entirely but its _target's_ `path.extname == ".la"`
+        # _could_ get handled here, but such a case is unlikely enough that this
+        # logic ignores it for now.)
```

Comment:
Thanks for the detailed comment. It probably also is not needed, since this hypothetical case likely will not arise. I suggest replacing lines 96 to 101 with `# Skip symlinks.`

Reply:
Skip the comment altogether; even `# Skip symlinks` is self-explanatory from the code.

Reply:
When the body of an if statement is empty, I tend to include a comment to indicate that it's left empty intentionally. Personal preference though. I'm fine with either.

Reply:
Yes, fair. Either is fine.